app/modules/conversations/conversations_router.py

```
@@ -194,8 +196,17 @@ async def post_message(
 
-            # Process images if present
-            attachment_ids = []
+            # Parse attachment_ids from form data if provided (e.g. pre-uploaded documents)
+            parsed_attachment_ids = []
+            if attachment_ids:
+                try:
+                    parsed_attachment_ids = json.loads(attachment_ids)
+                except json.JSONDecodeError:
+                    raise HTTPException(
+                        status_code=400, detail="Invalid attachment_ids format"
+                    )
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)